forch/forchestrator.py

```diff
@@ -149,16 +154,14 @@ def _initialize_faucetizer(self):
             structural_config = yaml.safe_load(file)
             self._faucetizer = faucetizer.Faucetizer(structural_config)
 
-        interval = self._config.get('orchestration', {}).get('faucetize_interval', 1)
-        dynamic_config_file = self._config.get('orchestration', {}).get(
-            'dynamic_config_file', _DYNAMIC_CONFIG_DEFAULT)
-        dynamic_config_path = os.path.join(os.getenv('FAUCET_CONFIG_DIR'), dynamic_config_file)
+        interval = self._config.get('orchestration', {}).get('faucetize_interval_sec', 60)
+        self._behavior_config_file = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), dynamic_config_file)
         self._faucetize_scheduler = HeartbeatScheduler(interval)
         self._faucetize_scheduler.add_callback(functools.partial(
```

Comment:
only one callback. Generally speaking you can't assume the ordering of callbacks, so if you have an atomic operation, it should be one callback.

Reply:
The callbacks are stored in a list in HeartbeatScheduler, which should gurantee the order. Do you mean the callbacks among different HeartbeatSchedulers?

Reply:
You only know that to be true because you know the internal implementation of the other class -- it's breaking the abstraction and bleeding details. If I were to change out the implementation of the scheduler, then that assertion might not be true.  So, yes, it _works_ in this circumstance, but it's _fragile_ and error prone.  Much better to code against the abstraction.   It's trivial to do -- you just create a lambda function and then call the two other functions from there.

Reply:
Done, PTAL